packages/TelegramClient-UI.package/TCUMessage.class/instance/messageDateText.st

```diff
@@ -0,0 +1,5 @@
+text conversion
+messageDateText
+
+	^ (String streamContents: [
+		:aStream | self date asTime print24: true showSeconds: false on: aStream ]), ' Uhr'
```

Comment:
Why leave this in German when the rest of the app is in english?

Reply:
Do you suggest just removing the `' Uhr'` or do we also want a 12h format? I found a method that displays this in a 12h format which is pretty convenient but I am not sur if that is what the customer wants.. What do you think?

Reply:
I suggest just removing the 'Uhr'. The official app does so as well in English